workflows/chipseq/chipseq-standard.wdl

```diff
@@ -133,8 +143,19 @@ workflow chipseq_standard {
         bam = markdup.mkdupbam,
         use_all_cores,
     }
+
     #@ except: UnusedCall
-    call picard.validate_bam { input: bam = markdup.mkdupbam }
+    call picard.validate_bam { input:
+        bam = markdup.mkdupbam,
+        ignore_list = [
+            "MISSING_PLATFORM_VALUE",
+            "INVALID_PLATFORM_VALUE",
+            "INVALID_MAPPING_QUALITY",
+            "MATES_ARE_SAME_END",
+            "MISMATCH_FLAG_MATE_NEG_STRAND",
+            "MISMATCH_MATE_ALIGNMENT_START"
```

Comment:
The first two seem "ok" to ignore. Less comfortable ignoring the last four. Are all these still being produced by seaseq?